frontend/rievent-frontend/src/components/Register.js

```diff
@@ -1,94 +1,153 @@
 import React, { useState } from 'react';
-import { Link, redirect } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
 import axios from 'axios';
-import Navbar from '../Navbar';
 import Cookies from 'js-cookie';
-import { useNavigate } from 'react-router-dom';
+import Navbar from '../Navbar';
+import { Typography, Button } from '@mui/material';
+import { makeStyles, styled, useTheme } from '@mui/styles';
+import { TextField } from '@mui/material/';
+
+const CustomTextField = styled(TextField)({
```

Comment:
CustomTextField and useStyles in separate file and the import. Use indentation of 4 spaces.